strings-csharp/RandomAlphaNumericString/RandomAlphaNumericString/Methods/SpanMethod.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Security.Cryptography;
+
+namespace RandomAlphaNumericString
+{
+    public partial class Methods
+    {
+        public static string? SpanMethod(int length)
+        {
+            Span<char> result = stackalloc char[length];
```

Comment:
Same as previously mentioned, this method could overflow the stack given a large length